airbyte-integrations/connectors/source-tweag-blog/integration_tests/sample_state.json

```diff
@@ -0,0 +1,5 @@
+{
+  "todo-stream-name": {
+    "todo-field-name": "value"
```

Comment:
Should it be changed/removed?

Reply:
Removed in https://github.com/tweag/airbyte/pull/2/commits/ad3cd0bf4105637948152fc7dccdf39c5ca5e6b5